project/project/plugins.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.6.0")
```

Comment:
I added this simply because it allows for checking for updates to SBT plugins in the SBT shell via
```
> reload plugins
> dependencyUpdates
```